src/examples/dialog/dialog-form.tsx

```diff
@@ -34,10 +44,21 @@ export class DialogFormExample {
         this.closeConfirmation = this.closeConfirmation.bind(this);
         this.onConfirmPositive = this.onConfirmPositive.bind(this);
         this.onConfirmNegative = this.onConfirmNegative.bind(this);
+        
+        this.scrollStop=this.scrollStop.bind(this);
     }
 
     public render() {
         return [
+            <limel-input-field
```

Comment:
We can probably remove this field
We should refactor the existing example instead and change the "age" input field in the dialog to `type="number"`